plugins/evict-pod.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: evict-pod
+spec:
+  version: v0.0.1
+  homepage: https://github.com/rajatjindal/kubectl-evict-pod
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-evict-pod/releases/download/v0.0.1/darwin-amd64-v0.0.1.tar.gz
+    sha256: 0bdcaca618c0304745c2813969111d87e3a476151d701ee06e55a88a28a4fc4a
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-evict-pod
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-evict-pod/releases/download/v0.0.1/linux-amd64-v0.0.1.tar.gz
+    sha256: 53f4cf2161921fce47e29b2db660088f075d2337dbbb790ec19be93fc475b025
+    files:
```

Comment:
Fyi i think 

From:* to:.

Is the default now. You don’t need to list explicitly.